docs/guides/governance/contributions/code-of-conduct/README.md

```diff
@@ -46,6 +50,19 @@ Access the standard Contributor Covenant template for a robust Code of Conduct p
 
 ---
 
+## Additional Guidance on Collaboration
+
+In addition to the standard Contributor Covenant, this project includes a supplementary document: [`CODE_OF_COLLAB.md`](https://raw.githubusercontent.com/NASA-AMMOS/slim/main/static/assets/governance/code-of-conduct/CODE_OF_COLLAB.md), which outlines expectations specific to scientific research collaboration.
+
+We recommend all contributors read both documents:
```

Comment:
Consider changing the wording here to be more specific about recommending the code of conduct for most, if not all soft projects, and the code of collab specifically for scientific software.